contributing_to_docs/doc_guidelines.adoc

```diff
@@ -911,22 +911,134 @@ kind: Pod
 metadata:
   labels:
     test: liveness
-...
+# ...
 ----
 +
 or
 +
 ----
-...
+# ...
 spec:
   containers:
   - args:
     image: k8s.gcr.io/liveness
-...
+# ...
 ----
 +
 Do not use `[...]`, `<snip>`, or any other variant.
 
+* Do not use `[source,yaml]` for the following:
```

Comment:
For this, I'm not sure why source,yaml is mentioned at all at the intro text. I would probably more actively say something like, be sure to use source,terminal for <this example> and source,text for <this example>